deploy-scripts/deploy_common.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+SLUG="kategory/kategory"
+JDK="oraclejdk8"
+BRANCH="master"
+
+set -e
+
+GRADLE_PROPERTIES_FILE=gradle.properties
+
+function getProperty {
+    PROP_KEY=$1
+    PROP_VALUE=`cat $GRADLE_PROPERTIES_FILE | grep "$PROP_KEY" | cut -d'=' -f2`
+    echo $PROP_VALUE
+}
+
+function fail {
+    echo "$1"
+    exit -1
+}
+
+VERSION_NAME=$(getProperty "VERSION_NAME")
```

Comment:
it looks odd to have constants, then functions, then constants

since this needs functions to be defined, how about separating constants and functions?